app/views/admin/notes/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :title, "Participant notes" %>
+<% content_for :before_content, govuk_back_link(text: "Back", href: admin_participant_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+   <span class="govuk-caption-l"><%= @participant_profile.user.full_name %></span>
+    <%= form_for @participant_profile, url: admin_note_path, method: :put do |f| %>
```

Comment:
I think if you used `form_for @participant_profile, as: :participant_profile` then you could avoid all the `profile_type` stuff in the `note_params` method and just do `params.require(:participant_profile).permit(:notes)`

Reply:
ahhhh ok, brill, yeah, that profile type is annoying, i'll switch it to your suggestion.